tests/test_fit.py

```diff
@@ -0,0 +1,53 @@
+import pandas as pd
+import pytest
+import sys
+import os
+from sklearn.datasets import load_diabetes
+from sklearn.model_selection import train_test_split
+from unittest.mock import Mock, patch
+
+
+
+#current_dir = os.path.dirname(os.path.abspath(__file__))
```

Comment:
This part is commented out. Do you wanna remove it if its not needed? Is there a reason why you would wanna leave it in?